platforms/hyperledger-besu/charts/besu_storage_class/templates/storageclass.yaml

```diff
@@ -0,0 +1,40 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "{{ .Values.metadata.name }}"
+reclaimPolicy: "{{ .Values.reclaimPolicy }}"
+volumeBindingMode: "{{ .Values.volumeBindingMode }}"
+{{- if eq .Values.cloud_provider "aws" }}
+provisioner: ebs.csi.aws.com
```

Comment:
make the provision parameterize should come from values.yaml, as it can have different values one

Reply:
Fixed.